report-to-row.ts

```diff
@@ -10,7 +10,8 @@ export const reportToRowHeaders = (csvFileContents: string) => {
       relax: true, // https://csv.js.org/parse/options/
     }
   );
-  if (!singleReportRows || singleReportRows.length === 0) return false;
+  if (!singleReportRows || singleReportRows.length === 0)
+    throw new Error('Unable to find report headers');
```

Comment:
Is there a way we might make this error more expressive? Is there something the user might have done to cause this?

Reply:
There is a comment on line 35 that might explain why this case is handled? I have never experienced the case where "urls come out half-baked"